lib/bootscale/utils.rb

```diff
@@ -0,0 +1,30 @@
+module Bootscale
+  module Utils
+    # Write to a file atomically. Useful for situations where you
+    # don't want other processes or threads to see half-written files.
+    #
+    #   Utils.atomic_write('important.file') do |file|
+    #     file.write('hello')
+    #   end
+    #
+    # Returns nothing.
+    def self.atomic_write(filename)
```

Comment:
Yeah sure. IMO it's already tested since if it was broken, pretty much all of the test suite would be as well. But I'd merge a PR with an unit test just for that.